src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -209,21 +209,21 @@ def create_chart_from_prometheus_query(
         label = get_target_name(series)
 
         if not label:
-            label = "\n".join([v for (key, v) in series.metric.items() if key != "job"])
+            label = "\n".join([v for (key, v) in series["metric"].items() if key != "job"])
```

Comment:
I'm not sure how this works
The `series_list_result` contains 2 different data types
On line 204, it gets a `List[dict]` , and on line 206 a `List[PrometheusSeries]`
PrometheusSeries isn't a dict, so how is this working?

Reply:
typing is unclear it is a dict, fixing